src/zac/js/utils/fetch.js

```diff
@@ -1,34 +1,38 @@
 const fetchDefaults = {
-    credentials: 'same-origin',  // required for Firefox 60, which is used in werkplekken
+    credentials: 'same-origin', // required for Firefox 60, which is used in werkplekken
 };
 
 const fetch = (url, opts) => {
-    const options = Object.assign({}, fetchDefaults, opts);
+    const options = { ...fetchDefaults, ...opts };
     return window.fetch(url, options);
 };
 
 const apiCall = fetch;
 
-
-const get = async (url, params={}) => {
+const get = async (url, params = {}, multiParams = []) => {
+    let searchParams;
     if (Object.keys(params).length) {
-        const searchparams = new URLSearchParams(params);
-        url += `?${searchparams}`;
+        searchParams = new URLSearchParams(params);
+    }
+    if (multiParams.length > 0) {
+        multiParams.map((param) => searchParams.append(
+            Object.keys(param)[0], param[Object.keys(param)[0]],
+        ));
```

Comment:
How about a bit more readable version? I had to really double-check with https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/append to figure out what exactly is happening in the code :) 

```suggestion
        multiParams.map((param) => {
            const paramName = Object.keys(param)[0]; // param={foo: bar}
            searchParams.append(paramName, param[paramName]);
        });
```